setup.py

```diff
@@ -568,8 +570,8 @@ def setup_paddle_extension() -> setuptools.Extension:
     else:
         if version >= (11, 2):
             nvcc_flags.extend(["--threads", "4"])
-        if version >= (11, 0):
-            nvcc_flags.extend(["-gencode", "arch=compute_80,code=sm_80"])
+        #if version >= (11, 0):
+        #    nvcc_flags.extend(["-gencode", "arch=compute_80,code=sm_80"])
```

Comment:
Why these changes?

Reply:
That was only for easy compilation on h100. Will remove them in my final cleanup.